### ISSUE TITLE
How to configure this project in IDE like Pycharm?

### ISSUE THREAD

**Issue by YaxuanZHOU** [TRI-ML/packnet-sfm#19]
Hi,
This is really great work. I'm trying to use this network on my data and I'm able to run the code in shell using the Makefile you provided. But i hope to run it in Pycharm so that debugging can be easier. However, since the project is packaged in Docker and also commands are written in the Makefile. I'm not sure how I can configure the project in Pycharm. Searched for many tutorials on line but couldn't find a working one maybe due to my limited understanding of Docker. 
I'm wondering if you have used IDE during the development of this project. Could you give some instructions on how to configure this project in Pycharm?
Best,

**Comment by VitorGuizilini-TRI** [comment]
Sorry, I am not familiar enough with Pycharm to help you with that. I will keep the issue open, maybe someone else can give you advice.

**Comment by fulkast** [comment]
Hey @YaxuanZHOU,

Have you checked out this guide here from PyCharm
https://www.jetbrains.com/help/pycharm/using-docker-as-a-remote-interpreter.html?

**Comment by YaxuanZHOU** [comment]
> Sorry, I am not familiar enough with Pycharm to help you with that. I will keep the issue open, maybe someone else can give you advice.

Thank you!

**Comment by YaxuanZHOU** [comment]
> Hey @YaxuanZHOU,
> 
> Have you checked out this guide here from PyCharm
> https://www.jetbrains.com/help/pycharm/using-docker-as-a-remote-interpreter.html?

Hi @fulkast , thanks for this link. I browsed it and I did the configuration of docker interpreter and also docker service before. After these steps, I tried to run the train.py script on the kiti-tini data, but the error shows that there are some libraries not installed and can't be imported, which indicates that I may not be running the docker container correctly or I'm not running the code in the packnet-sfm docker container at all. I'm guessing the last missing part of the configuration might be that I need to put the things that's in the Makefile (which defines a lot of arguments that's needed when running the docker container as desired) into Pycharm. But I'm not sure how to do that.

Thanks!
